examples/plot_rz_slices_ring_source.py

```diff
@@ -69,6 +69,16 @@
 my_tally_result = statepoint.get_tally(name="my_tally")
 
 for slice_index in range(1, len(mesh.phi_grid) - 1):
+    plot = plot_mesh_tally_rz_slice(
+        tally=my_tally_result,
+        geometry=my_geometry,
+        outline=True,
+        norm=LogNorm(),
+        slice_index=slice_index,
+        reflect_in_z=True
```

Comment:
```suggestion
        mirror=True
```

Reply:
the testing in the CI caught this one that we missed when changing from reflect_in_z to mirror

Reply:
I have just pushed the update to the brach - good catch from the CI!